packages/react-devtools-extensions/src/background/dynamicallyInjectContentScripts.js

```diff
@@ -6,15 +6,15 @@ const contentScriptsToInject = [
     js: ['build/proxy.js'],
     matches: ['<all_urls>'],
     persistAcrossSessions: true,
-    runAt: 'document_end',
+    runAt: 'document_start',
     world: chrome.scripting.ExecutionWorld.ISOLATED,
   },
   {
     id: '@react-devtools/file-fetcher',
     js: ['build/fileFetcher.js'],
     matches: ['<all_urls>'],
     persistAcrossSessions: true,
-    runAt: 'document_end',
```

Comment:
`file-fetcher` script is probably best to keep initialized at `document_end`, no real need for it this early

Reply:
Yes, but only as a last-resort option. The browser extension is using browser extensions API first to get resources from the page and only then falls back to the fetch call. To be fair, I am not even sure this still works, we are just relying that the resource is still cached somewhere in `fetch` implementation.